packages/@expo/cli/src/start/doctor/dependencies/getVersionedPackages.ts

```diff
@@ -90,6 +90,14 @@ export async function getRemoteVersionsForSdkAsync({
   }
 }
 
+type ExcludedNativeModules = {
+  name: string;
+  bundledNativeVersion: string;
+  hasSpecifiedExactVersion: boolean;
+  isExcludedFromValidation: boolean;
+  specifiedVersion?: string; // e.g. 1.2.3, latest
```

Comment:
could `hasSpecifiedExactVersion` and `specifiedVersion` be collapsed into a single property? (given that you should be able to derive the former from the latter)

Reply:
Done. It wasn't used anywhere.